backToCart/index.ts

```diff
@@ -36,12 +35,24 @@ export default class BackToCart {
      */
     onMenuItemClick(): () => void {
         return () => {
+            if (!this.getCatalogueUrl()) {
+                return;
+            }
             // save bookmark in local storage so it is restored when user returns
             sessionStorage.setItem(this.api.id, this._RV.getBookmark());
             window.location.href = this.getCatalogueUrl();
         };
     }
 
+    /**
+     * Auto called by RAMP startup, checks for a catalogueUrl in the config and sets the template if so
+     */
+    preInit(pluginConfig: any): void {
```

Comment:
Define the `pluginConfig` interface instead of using `any` type.

Reply:
Done